fence/agents/pve/fence_pve.py

```diff
@@ -142,8 +142,19 @@ def main():
 			"(Optional, will be automatically determined)",
 		"order": 2
 	}
+	all_opt["vm_type"] = {
```

Comment:
Can you change it to vmtype here as well (and in the device_opt line) for consistency? Remember to do "make xml-upload" as well afterwards.